include/SDL_render.h

```diff
@@ -1414,6 +1423,51 @@ extern DECLSPEC int SDLCALL SDL_RenderCopyExF(SDL_Renderer * renderer,
                                             const SDL_FPoint *center,
                                             const SDL_RendererFlip flip);
 
+/**
+ *  Render a list of triangles, optionally using a texture and indices into the vertex array
+ *  Color and alpha modulation is done per vertex (SDL_SetTextureColorMod and SDL_SetTextureAlphaMod are ignored).
+ *
+ *  \param texture      (optional) The SDL texture to use.
+ *  \param vertices     Vertices.
+ *  \param num_vertices Number of vertices.
+ *  \param indices      (optional) An array of integer indices into the 'vertices' array, if NULL all vertices will be rendered in sequential order.
+ *  \param num_indices  Number of indices.
+ *
+ *  \sa SDL_Vertex
+ *
+ *  \return 0 on success, or -1 if the operation is not supported
+ */
+extern DECLSPEC int SDLCALL SDL_RenderGeometry(SDL_Renderer *renderer,
+                                               SDL_Texture *texture,
+                                               const SDL_Vertex *vertices, int num_vertices,
+                                               const int *indices, int num_indices);
+
+/**
+ *  Render a list of triangles, optionally using a texture and indices into the vertex arrays
+ *  Color and alpha modulation is done per vertex (SDL_SetTextureColorMod and SDL_SetTextureAlphaMod are ignored).
+ *
+ *  \param texture      (optional) The SDL texture to use.
+ *  \param xy           Vertex positions
+ *  \param xy_stride    Byte size to move from one element to the next element
+ *  \param color        Vertex colors (as SDL_Color)
+ *  \param color_stride Byte size to move from one element to the next element
+ *  \param uv           Vertex normalized texture coordinates
+ *  \param uv_stride    Byte size to move from one element to the next element
+ *  \param num_vertices Number of vertices.
+ *  \param indices      (optional) An array of indices into the 'vertices' arrays, if NULL all vertices will be rendered in sequential order.
+ *  \param num_indices  Number of indices.
+ *  \param size_indice  Indice size: 1 (byte), 2 (short), 4 (int)
+ *
+ *  \return 0 on success, or -1 if the operation is not supported
+ */
+extern DECLSPEC int SDLCALL SDL_RenderGeometryRaw(SDL_Renderer *renderer,
+                                               SDL_Texture *texture,
+                                               const float *xy, int xy_stride,
```

Comment:
I'd expect these parameters to use the same types as the ones in the `SDL_Vertex` struct (eg an array of `SDL_Color` instead of an array of `int`).

Reply:
SDL_Color are int anyway. 
SDL_RenderGeometryRaw is added for app that are non SDL (eg: librocket, dear imgui, nuklear, spine).
So it's better to present them a basic type than something they don't know, they will need to look for, and that are going to cast anyway.

If you want to draw triangles with an SDL app, more friendly to use SDL_Vertex and SDL_RenderGeometry.